testsuite/MDAnalysisTests/test_selections.py

```diff
@@ -61,6 +61,9 @@ def _write_selection(self, **kwargs):
         g.write_selection(self.namedfile, **kwargs)
         return g
 
+    def test_write_bad_mode(self):
```

Comment:
That code has already been merged into develop, see: https://github.com/MDAnalysis/mdanalysis/pull/989